src/sidebar/components/Excerpt.tsx

```diff
@@ -95,19 +113,19 @@ function Excerpt({
   onToggleCollapsed = noop,
   overflowThreshold = 0,
   settings = {},
-}) {
+}: ExcerptProps) {
   const [collapsedByInlineControls, setCollapsedByInlineControls] =
     useState(true);
 
-  const contentElement = /** @type {{ current: HTMLDivElement }} */ (useRef());
+  const contentElement = useRef<HTMLDivElement>();
```

Comment:
```suggestion
  const contentElement = useRef<HTMLDivElement|null>(null);
```

This should fix your issue with setting `ref` on `div[data-testid="excerpt-content"]`